db/migrate/20230224042902_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] According to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), kindly ensure that a `Like` belongs to an `:author` and not a `:user` so you should  refer the user table for the `foreign_key`. i.e.
```ruby
  def change
    add_reference :likes, :author, null: false, foreign_key: `{ to_table: 'users' }`
  end
```